### PR TITLE
Add JS DOM utils to Scaladoc-js

### PR DESCRIPTION
**Pull request by pikinier20**

### CODE REVIEW

File: scaladoc-js/common/src/utils/html.scala [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  val defer = Attr("defer")
+  val src = Attr("src")
+  val rel = Attr("rel")
+  val charset = Attr("charset")
+  val name = Attr("name")
+  val content = Attr("content")
+  val testId = Attr("data-test-id")
+  val alt = Attr("alt")
+  val value = Attr("value")
+  val onclick=Attr("onclick")
```
Comment (MarcinAman) [inline scaladoc-js/common/src/utils/html.scala:112]: format

Comment (MarcinAman) [reply scaladoc-js/common/src/utils/html.scala:112]: also: i'd use onClick

File: scaladoc-js/common/src/utils/html.scala [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  val table = Tag[domhtml.Table]("table")
+  val thead = Tag[domhtml.Element]("thead")
+  val tbody = Tag[domhtml.Element]("tbody")
+  val th = Tag[domhtml.TableCell]("th")
+  val tr = Tag[domhtml.TableRow]("tr")
+  val td = Tag[domhtml.TableCell]("td")
+  val b = Tag[domhtml.Element]("b")
+  val i = Tag[domhtml.Element]("i")
+
+  val cls = Attr("class")
```
Comment (MarcinAman) [inline scaladoc-js/common/src/utils/html.scala:97]: class, we do not code in C++ to count every character we type

**Review by BarkingBad — APPROVED** [review]

File: scaladoc-js/common/src/utils/html.scala (hunk 2 above)
Comment (pikinier20) [reply scaladoc-js/common/src/utils/html.scala:97]: `class` is a keyword and I didn't want to type:
```
`class`
```
in every tag

**Pull request merged**